projects/lz4/project.yaml

```diff
@@ -5,6 +5,7 @@ auto_ccs:
   - "cyan@fb.com"
   - "terrelln@fb.com"
   - "felixh@fb.com"
+  - "bimbashrestha@fb.com"
```

Comment:
Probably all of the FB folks should go in vendor_ccs ? See docs here 
https://google.github.io/oss-fuzz/getting-started/new-project-guide/#vendor

Reply:
Thanks for the tip! All these FB people are directly on or related to the team, so they want full access. Will keep `vendor_ccs` in mind for the future.

Reply:
Yes sounds good. just fyi, vendor_ccs gives same access, but if you are part of dev team, then auto_ccs is the section to go.